python/desc/monitor/monitor.py

```diff
@@ -110,6 +116,54 @@ def build_lightcurve(self, objid):
                     lightcurve['zpsys'].append('ab')
         self.lightcurve = Table(data=lightcurve)
 
+    def build_lightcurve_from_db(self, objid=None, ra_dec=None, tol=0.005):
```

Comment:
Here the change would be
```
    def build_lightcuve_from_db(self, objid=None, ra_dec=None, tol=0.005, database='DESC_Twinkles_Level_2',
                                                  host='scidb1.nersc.gov', port=3306, driver='mysql'):
```
and the last four would be passed to the `dbi(...)` call below.   It's a little inconvenient to provide the db details with every call to `.build_lightcurve_from_db`, which is one of the reasons I decided to use the Factory pattern in my implementation of `LightCurveFactory` in Twinkles.   We might consider doing something like that here.  Also, the Factory pattern makes it more natural, I think, to serve up immutable `LightCurve` objects so that the light curve data can be provided through the constructor or a private method.  There may be some db connection management issues associated with creating the connection objects locally like you're doing here.  Those concerns could be mitigated again with the Factory pattern.

In the near term, it's probably fine to make the minimal changes so that we can merge this into master and people can start using it.